dipy/reconst/fwdti.py

```diff
@@ -344,7 +344,7 @@ def wls_iter(
 
 
 def wls_fit_tensor(
-    gtab, data, Diso=3e-3, mask=None, min_signal=1.0e-6, piterations=3, mdreg=2.7e-3
+    gtab, data, mask=None, min_signal=1.0e-6, piterations=3, mdreg=2.7e-3
```

Comment:
Hi @jhlegarreta,

Diso should not be removed but it should be used here:

https://github.com/dipy/dipy/blob/8be51800309373135cdda26bbb7bfd9d310c1d41/dipy/reconst/fwdti.py#L423

Can you update this, then it is ready to go. Thanks!

Reply:
:+1: Nice catch. Done.